src/crypto-api/index.ts

```diff
@@ -555,21 +552,24 @@ export interface CryptoApi {
     deleteKeyBackupVersion(version: string): Promise<void>;
 
     /**
-     * Download the last key backup from the homeserver (endpoint GET /room_keys/keys/).
+     * Download and restore the last key backup from the homeserver (endpoint GET /room_keys/keys/).
      * The key backup is decrypted and imported by using the decryption key stored locally. The decryption key should be stored locally by using {@link CryptoApi#storeSessionBackupPrivateKey}.
```

Comment:
If we deprecate `storeSessionBackupPrivateKey`, how we can set the recovery key when it filled by the user?

In EW when restoring key backup, if we can't restore with cache or SSSS, we ask for the recovery key from the user

Reply:
I guess you mean the backup decryption key, rather than the recovery key.

I'm not sure we need to support the user entering the backup decryption key directly -- I thought most of that went away with https://github.com/element-hq/element-web/issues/27455.

But sure, let's leave this for now.